package-lock.json

```diff
@@ -1,8 +1,15684 @@
 {
```

Comment:
I would prefer not to have the package-lock.json in the PR. I generate that only on new releases. Also please direct your PR to mailvelope:dev and not to master.